draft-ietf-aipref-vocab.md

```diff
@@ -203,11 +203,144 @@ If arrangements, such as contracts exist that explicitly permit the use of that
 
 The vocabulary does not preclude the use of other specific categories. Any statement of preference based on this vocabulary shall not be interpreted as restricting the use of the work(s) strictly for the purpose of search and discovery as long as no restriction is declared through search-specific means such as {{!RFC9309}}.
 
-## Vocabulary Extensions
+## Vocabulary Extensions {#vocab-extension}
 
 Systems referencing the vocabulary must not introduce additional categories that include existing categories defined in the vocabulary or otherwise include additional hierarchical relationships.
 
 
+# Serialization Format {#format}
+
+This section defines an exemplary serialization format for preferences.
+The format describes how the abstract model could be turned into a Unicode string or sequence of bytes.
+
+The format relies on the Dictionary type defined in {{Section 3.2 of !FIELDS=RFC9651}}.
+The dictionary keys correspond to usage categories
+and the dictionary values correspond to explicit preferences.
+
+For example, the following is a preference to allow AI training ({{ai}}),
+disallow generative AI training ({{genai}}), and
+and state no preference for other categories other than subsets of these categories:
+
+~~~
+ai=y, genai=n
+~~~
+
+
+## Usage Category Labels
+
+Each usage category in the vocabulary ({{vocab}}) is mapped to a short textual label.
+{{t-category-labels}} tabulates this mapping.
+
+| Category               | Label | Reference |
+|:-----------------------|:------|:----------|
+| Text and Data Mining   | tdm   | {{tdm}}   |
+| AI Training            | ai    | {{ai}}    |
+| Generative AI Training | genai | {{genai}} |
+{: #t-category-labels title="Mappings for Categories"}
+
+Any mapping for a new usage category can only use
+lowercase latin characters (a-z), digits (0-9), "_", "-", ".", or "*".
+These are encoded using the mappings in {{!ASCII=RFC0020}}.
+
+
+## Preference Labels
+
+The abstract model used has two options for preferences associated with each category:
```

Comment:
at this time - unless we add a third in the vocab.

And if we do, how would that be serialized?